crates/runtime/src/dataaccelerator/postgres.rs

```diff
@@ -100,6 +114,26 @@ impl DataAccelerator for PostgresAccelerator {
 
         let ctx = SessionContext::new();
 
+        // Validate and normalize pool_min and connection_pool_size
+        let connection_pool_min_idle = cmd
+            .options
+            .get("connection_pool_min")
```

Comment:
nit: if user provides invalid `connection_pool_min`, they will not be warned that it is being ignored for a default value

Reply:
Good point, I'll fix.